.readthedocs.yaml

```diff
@@ -5,17 +5,18 @@
 # Required
 version: 2
 
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
```

Comment:
move version to build.tools as per the doc https://docs.readthedocs.io/en/stable/config-file/v2.html